src/machine.rs

```diff
@@ -267,6 +269,9 @@ impl<'m> Machine<'m> {
         self.send_action(Action::SendCtrlAltDel).await?;
         trace!("{vm_id}: CTRL+ALT+DEL sent to VM successfully.");
 
+        // Give change to the machine to shutdown cleanly before removing its files.
+        sleep(Duration::from_secs(10)).await;
```

Comment:
if we sleep here then we should probably remove the sleep in `delete`

Reply:
oh, actually I forgot to delete this hunk.